advanced/wallets/react-wallet-v2/src/lib/SolanaLib.ts

```diff
@@ -78,8 +78,22 @@ export default class SolanaLib {
     return { signature }
   }
 
+  public async signAllTransactions(
+    params: SolanaLib.SignAllTransactions['params']
+  ): Promise<SolanaLib.SignAllTransactions['result']> {
+    const signedTransactions = params.transactions.map(transaction => {
+      const transactionObj = this.deserialize(transaction)
+
+      this.sign(transactionObj)
+
+      return this.serialize(transactionObj)
+    })
+
+    return { transactions: signedTransactions }
+  }
+
   private serialize(transaction: VersionedTransaction): string {
-    return bs58.encode(transaction.serialize())
+    return Buffer.from(transaction.serialize()).toString('base64')
```

Comment:
won't this break current integrations ?

Reply:
The only integration using serialized transactions right now comes from AppKit, which have been changed to `base64`, the main integrations are using `signature` param which don't pass through transaction serialization.